.github/workflows/ci-bun-transpiler.yml

```diff
@@ -18,8 +18,7 @@ jobs:
         with:
           bun-version-file: .tool-versions
       - run: yarn workspaces focus hono-middleware @hono/bun-transpiler
-      - run: yarn workspace @hono/bun-transpiler build
-      - run: yarn workspace @hono/bun-transpiler publint
+      - run: yarn build --filter bun-transpiler
```

Comment:
`tsdown` also integrates `arethetypeswrong` and `publint` so we can remove the `publint` step ✨